Update feature-flags.md (#223)
ember-learn/guides-source: guides/v3.5.0/configuring-ember/feature-flags.md

the canary and future beta branches, and the feature becomes part of the framework.

## Flagging Details

> The flag status in the generated build is controlled by the `features.json`

Reviewer: ```suggestion
The flag status in the generated build is controlled by the [`@ember/canary-features`](https://github.com/emberjs/ember.js/blob/master/packages/@ember/canary-features/index.ts)
```

> file in the root of the Ember.js project. This file lists all new features and their current status.

Reviewer: ```suggestion
package. This package exports a list of all available features and their current status.
```
